spdx-utils/src/main/kotlin/model/SpdxConstants.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.spdx.model
+
+object SpdxConstants {
+    /**
+     * Represents a not present value, which has been determined to actually be not present. This representation must
+     * not be used if [NOASSERTION] could be used instead.
+     */
+    const val NONE = "NONE"
+
+    /**
+     * Represents a not present value where any of the following cases applies:
+     *
+     * 1. no attempt was made to determine the information.
+     * 2. intentionally no information is provided, whereas no meaning should be derived from the absence of the
+     *    information.
+     */
+    const val NOASSERTION = "NOASSERTION"
```

Comment:
I deliberately decided against SpdxLicense since these constants are not exclusively used for license  properties but also for other ones. I'd rather re-use the SPDXConstants from the SPDXLicense, maybe moving the SPDXConstants class out of the model package, one level up the tree.

Reply:
> since these constants are not exclusively used for license properties

If that's the case I'd actually drop them from `SpdxLicense` instead, and make existing code use the constants from `SpdxConstants` (which could in the same run be moved to a more central place). But I'm fine doing this as a follow-up PR.